src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -350,6 +350,8 @@ std::map<size_t, memory::ptr> program_node::get_const_memory_deps() const {
 void program_node::invalidate_users() const {
     for (auto& user : users) {
         if (user->valid_output_layout) {
+            if (user->is_type<convolution>() && user->as<convolution>().get_required_output() != format::any)
```

Comment:
Couldn't this condition be somehow generalized? And I'm afraid it may break sequential users invalidation - did you check the effects of this?